pkg/cloud/azure/assets/cloud-controller-manager-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: azure-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    app: azure-cloud-controller-manager
+spec:
+  selector:
+    matchLabels:
+      app: azure-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: azure-cloud-controller-manager
+    spec:
+      hostNetwork: true
+      serviceAccountName: cloud-controller-manager
+      priorityClassName: system-cluster-critical
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
```

Comment:
does this also need to tolerate `not-ready` ?

seems like aws does, https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/pkg/cloud/aws/assets/deployment.yaml#L55

Reply:
None of our components should be tolerating that taint, if we do, we will schedule onto a node without networking, which will likely cause our pods to crash loop as they can't contact the APIs they need to contact

Reply:
It will now need to tolerate the not-ready based on the additions made in #76